### PR TITLE
Update to docs for building docs

### PR DESCRIPTION
**Pull request by ehsteve**
Current docs are out of date on describing how to build docs locally.

### CODE REVIEW

File: doc/source/dev.rst [hunk 1]
```diff
@@ -15,29 +15,29 @@ Version Control
...
 
 **Getting Started**
 
-Creating your own repo on GitHub is easy to do. If you followed the SunPy installation
-instructions you should already have git installed. Go ahead and create a free account
-on `create an account on GitHub <https://github.com/signup/free>`_. Github has some
-`great resources to help <https://help.github.com/>`_. Here is a quick overview of the
-process.
+Creating your own fork of SunPy on GitHub is easy to do. Go ahead and create
+a free accountvon `create an account on GitHub <https://github.com/signup/free>`_.
```
Comment (nabobalis) [inline doc/source/dev.rst]: accountvon -> account on

File: doc/source/dev.rst [hunk 2]
```diff
@@ -94,34 +96,43 @@ as a `remote` in git: ::
...
  git commit -a -m "My first commit"
  git push
 
-You local repo is now synced with GitHub and ahead of the main repo as it contains
-your personal contribution. Remember to commit after you've done a unit of work (i.e.
-often). This will make it easier for you (in the future) and everyone else to understand
-what you are doing. Also make sure to make your commit statements clear and understandable.
+You local repo is now synced with GitHub and ahead of the main repo as it
+contains your change. Remember to commit after you've done a unit of work (i.e.
+often). This will make it easier for you (in the future) and everyone else to
```
Comment (nabobalis) [inline doc/source/dev.rst]: I think this sets up the wrong idea. We do not want PRs with each time a file or line change to be committed.

Comment (Cadair) [inline doc/source/dev.rst]: disagree, we should encourage commit early commit often (it's good git practice), we can always squash merge.

File: docs/dev.rst [hunk 3]
```diff
@@ -442,23 +453,25 @@ Sphinx.
...
+``examples``. To build the documentation locally you must have Sphinx
+(as well as Numpydoc, astropy-helpers, and sphinx-gallery) installed on
+your computer. In the root directory run ::
 
-This will generate HTML documentation for SunPy. To clean up and delete the
-generated documentation run: ::
+    python setup.py build_docs
 
-    make clean
+This will generate HTML documentation for SunPy in the ``build/sphinx/html``
```
Comment (Cadair) [inline docs/dev.rst]: @nabobalis this is no longer correct right?

Comment (nabobalis) [inline docs/dev.rst]: It is now _build

**Comment by Cadair** [comment]
Test fails are due to HEK being offline.

**Pull request merged**